File: lalframe/swig/python/check-swiglalframe.py
# check SWIG Python module wrapping the LALFrame library
# Author: Karl Wette, 2011

import os

def msg(str):
    print(os.path.basename(__file__) + ": " + str)

# check module load
import swiglalframe
from swiglalframe import *
msg("passed module load")

# passed all tests!
msg("================")
msg("PASSED all tests")
msg("================")

File: lalframe/swig/octave/check-swiglalframe.m
## check SWIG Octave module wrapping the LALFrame library
## Author: Karl Wette, 2011

function msg(str)
  disp([program_name, ": ", str]);
endfunction

## check module load
swiglalframe;
msg("passed module load");

## passed all tests!
msg("================");
msg("PASSED all tests");
msg("================");
